.github/workflows/unclaim-cf-env.yml

```diff
@@ -0,0 +1,43 @@
+name: Unclaim an environment
+
+on:
+  workflow_call:
+    inputs:
+      environment:
+        required: true
+        type: string
+      identifier-for-metadata-file:
+        required: true
+        type: string
+
+jobs:
+  build-env:
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+
+    steps:
+      - name: Download metadata
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{ inputs.identifier-for-metadata-file }}
```

Comment:
some testing in a private space seems to indicate that this line is not quite correct.  I'm getting a problem where this input is empty (`''`) causing all artifacts to be downloaded, which results in a pathing problem.  For example, if the environment name is `rosebud` then instead of downloading just the metadata json file, it puts in the json file in a directory called `rosebud`.  This causes the next step to fail on `cat metadata.json | jq -r .name` as the file is actually `rosebud/metadata.json`

I'm hoping I'm just not seeing something silly, like the input is badly passed in `integration.yml` or incorrectly exported out `setup-cf-env.yml`

[github docs](https://docs.github.com/en/actions/using-workflows/reusing-workflows#using-outputs-from-a-reusable-workflow) for reference on anyone trying to help

Reply:
Ok thinking harder for a few minutes, the problem is actually over [here](https://github.com/cloudfoundry/cli/pull/2326/files#r986041041)